config/purposes/heron.yml

```diff
@@ -7,8 +7,18 @@ LHR RT:
   :default_printer_type: :plate_a
   :label_template: plate_lhr_rt
   :presenter_class: Presenters::StockPlatePresenter
+LHR PCR 1:
+  :asset_type: plate
+LHR PCR 2:
+  :asset_type: plate
```

Comment:
Oh nice, so you didn't need a new creator for the PCR plates, because they are the same as each other, they will just have 2 buttons to click?